src/DocumentFormat.jl

```diff
@@ -18,7 +18,8 @@ struct FormatOptions
     comments::Bool
     docs::Bool
     lineends::Bool
-    kw::Bool
+    keywords::Bool
+    kwarg::Int # Options arg (0 => no spacing around kw assignment, 1 => single space around kw assignment, anything else => off)
```

Comment:
Can we use a different type here? Symbol? Or at least an enum?

Reply:
Sure